packages/starlight/__tests__/sidebar/navigation-hidden.test.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, expect, test, vi } from 'vitest';
+import { getSidebar } from '../../utils/navigation';
+
+vi.mock('astro:content', async () =>
+	(await import('../test-utils')).mockedAstroContent({
+		docs: [
+			['index.mdx', { title: 'Home Page' }],
+			['environmental-impact.md', { title: 'Eco-friendly docs' }],
+			['reference/configuration.md', { title: 'Config Reference' }],
+			['reference/frontmatter.md', { title: 'Frontmatter Reference', sidebar: { hidden: true } }],
+			['guides/components.mdx', { title: 'Components' }],
+		],
+	})
+);
+
+describe('getSidebar', () => {
+	test('returns sidebar entries sorted by frontmatter order', () => {
```

Comment:
Test name needs updating here:

```suggestion
	test('excludes sidebar entries with hidden: true in frontmatter', () => {
```